code/__defines/turfs.dm

```diff
@@ -2,11 +2,14 @@
 #define TURF_REMOVE_SCREWDRIVER 2
 #define TURF_REMOVE_SHOVEL      4
 #define TURF_REMOVE_WRENCH      8
-#define TURF_CAN_BREAK          16
-#define TURF_CAN_BURN           32
-#define TURF_HAS_EDGES          64
-#define TURF_HAS_CORNERS        128
-#define TURF_IS_FRAGILE         256
-#define TURF_ACID_IMMUNE        512
-#define TURF_USE0ICON           1024
-#define SMOOTH_ONLY_WITH_ITSELF 2048
\ No newline at end of file
+#define TURF_REMOVE_WELDER		16
+#define TURF_CAN_BREAK          32
+#define TURF_CAN_BURN           64
+#define TURF_HAS_EDGES          128
+#define TURF_HAS_CORNERS        256
+#define TURF_IS_FRAGILE         512
+#define TURF_ACID_IMMUNE        1024
+#define TURF_USE0ICON           2048
+#define SMOOTH_ONLY_WITH_ITSELF 4096
+#define TURF_HIDES_THINGS		8192
```

Comment:
Hides things flag indicates that the turf will hide pipes and wires underneath it. Without this flag, it won't.